### ISSUE TITLE
The Cobra Effect

### ISSUE THREAD

**Issue by dwmkerr** [dwmkerr/hacker-laws#291]
https://en.wikipedia.org/wiki/Cobra_effect

This is an interesting one, in some ways it ties into Gall's Law and "Reward Hacking".

**Comment by umutphp** [comment]
Yes, this should be in the document :). Any software developer has already created a cobra effect at least once in one of his/her projects.
